src/math/Statistics.cc

```diff
@@ -337,6 +350,7 @@ StandardReturn processPixels(ImageT const &img, MaskT const &msk, VarianceT cons
  * @param weightsAreMultiplicative weights are multiplicative (not inverse)
  * @param andMask mask of bad pixels
  * @param calcErrorFromInputVariance estimate errors from variance
+ * @param calcErrorMosaicMode estimate errors by taking mean of input variances
```

Comment:
Rather than change the name from "mosaicMode", which is as good as anything I can come up with right now, let's just expand the docs here to connect the name to the calculation, e.g. "estimate errors by taking the mean of input variances, as is appropriate when those variances are totally correlated.  This is the case when mosaicking coadds built from (more or less) the same input images".